scripts/generate_stats.py

```diff
@@ -73,6 +73,10 @@ def traffic_stats(config, ranges):
 def ga_stats(config, ranges):
     stats = {}
 
+    # We currently sample google analytics sessions, so we need to
+    # multiply to determine the true visitor count. e.g. '50' for 50%.
+    SAMPLE_MULTIPLIER = 100 / int(g.googleanalytics_sample_rate)
```

Comment:
Should this be a float division in case the sample rate doesn't cleanly divide into 100?